source/client-side-encryption/client-side-encryption.rst

```diff
@@ -1114,28 +1114,28 @@ easier for users to distinguish when a command fails due to client side
 encryption.
 
 
-.. index:: csfle
-.. _Enabling csfle:
+.. index:: crypt_shared
+.. _Enabling crypt_shared:
 
-Enabling Command Marking with the csfle_ Library
-=================================================
+Enabling Command Marking with the crypt_shared_ Library
+=======================================================
 
 The MongoDB Enterprise distribution includes a dynamic library named
-``mongo_csfle_v1`` (with the appropriate file extension or filename suffix for
+``mongo_crypt_v1`` (with the appropriate file extension or filename suffix for
```

Comment:
Is the library called `crypt` and not `crypt_shared`?

If so, I'm confused as to why all of the references are to "crypt_shared".